src/modules/ui.js

```diff
@@ -60,6 +60,18 @@ class CoreUiApi extends BaseApi {
     hideHelp() {
         super.sendMsgToPc('hideHelp');
     }
+
+    /**
+     * Show an interaction by ID.
```

Comment:
Thinking of moving this out to a conversations.js module?  This will mirror the platform api which will facilitate users already familiar with it.  Also, the showUserDetails is already in a separate users.js module so it would be consistent there.

Thoughts?

Reply:
Yea we should keep it consistent with the api.